hack/build-manifests.sh

```diff
@@ -30,23 +30,13 @@ trap 'catch $? $LINENO' ERR TERM INT
 # Lastly, we take give the component CSVs to the csv-merger that combines all
 # of the manifests into a single, unified, ClusterServiceVersion.
 
-function get_image_digest() {
-  if [[ ! -f ${PROJECT_ROOT}/tools/digester/digester ]]; then
-    (
-      cd "${PROJECT_ROOT}/tools/digester"
-      go build .
-    )
-  fi
-
-  local image
-  image=$("${PROJECT_ROOT}/tools/digester/digester" -image "$1" "$2")
-  echo "${image}"
-}
-
 PROJECT_ROOT="$(readlink -e $(dirname "${BASH_SOURCE[0]}")/../)"
 source "${PROJECT_ROOT}"/hack/config
 source "${PROJECT_ROOT}"/deploy/images.env
 
+# update image digests
+"${PROJECT_ROOT}"/automation/digester/update_images.sh
```

Comment:
This should be above `source "${PROJECT_ROOT}"/deploy/images.env` (but still below `source "${PROJECT_ROOT}"/hack/config`)